x-pack/plugins/fleet/server/routes/message_signing_service/handlers.ts

```diff
@@ -41,6 +41,6 @@ export const rotateKeyPairHandler: FleetRequestHandler<
     });
   } catch (error) {
     logger.error(error.meta);
```

Comment:
should we log the whole error instead of `error.meta` here?

Reply:
Yeah, why not. I'll update it.

Reply:
done 723461e9920f7ab9a948ab0f750e8a469f51af6d